### ISSUE TITLE
Some subscriptions for YouTube are not showing

### ISSUE THREAD

**Issue by 2020tmzg** [TeamNewPipe/NewPipe#9349]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

Some subscriptions cannot display the latest programs. For example, my subscription to 大康有话说 can only display programs from a month ago.

**Comment by mycation** [comment]
I have the same problem. On YouTube Most channels have no videos or only a few videos or lots of missing videos. The missing videos have no consistent pattern I can see related to time. Some channels have a video posted within the hour and missing some posted hours ago. I can't see any live streaming videos or channels, none have any live streaming visible.

Also I can't find the videos that are missing from the channel list by searching for them in search. They seem to be missing from searching also. I haven't exhausted all searches but the ones that I've tried to find via search, I can't find.

**Comment by TobiGr** [comment]
@Master2050 You have been repeadadly violated our contribution guidelines by creating many duplicate issues and much traffic ingeneral. Make sure to stick to our contribution guidelines when contributing to our projects. We have requested you to do so several times in the last two weeks. Our time is limited (this is our spare time and we are doing it voluntarily) and we need allcontributors to stick to a basic set of rules. Otherwise, this project will become more of a burden than a fun project for us maintainers.
We blocked your account temporaray for the next 7 days. We encourage you to read the contribution guidelines completely. This is generally mandatory before starting to contribute in any way. We are happy about any contributions that bring value to the project.

**Comment by opusforlife2** [comment]
Copying my comment here for context:

See https://github.com/TeamNewPipe/NewPipe/issues/9349#issuecomment-1320317474 for the reason you got a temp ban, Master2050.

> I have never promoted, endorsed or posted any demeaning, abusive, sexual, offensive or action taken by mere malice

Correct.

> No warning were issued by anyone nor any discussion occurred in the aforesaid regard

**Wrong.** You were warned several times, by multiple people. You just seemed to ignore it, which is why a drastic action had to be taken. Since you have read access to the repo, you can easily read these warnings again. Maybe that would help refresh your memory.

> I have always acted in good faith

It stops being in good faith when you ignore what maintainers are trying to tell you about your behaviour. To use a metaphor, you can't punch people in the face, and then say you're a good person because you help old people cross the street. You have to stop punching people too. You weren't doing that despite us telling you.

> I have made singnificant contribution

In general, yes, we do value community members who help other users out, and your contributions are welcome in this regard, but again, the record is spoiled when you keep 'punching people'.

> I have invested significant amount of time in aforesaid actions.

We would love it if you invest less time in opening duplicate issues, and more time in searching for them. When you joined the IRC channel, we were all happy thinking that the problem would stop and you would interact with us directly, which would prevent spamming our notifications. But you never used it and went back to doing what you were doing. Back to square one.

> What i won't do from now onwards?

That is up to you. Your time is your own.

> Reason: Hostile behaviour towards me

This has been explained sufficiently now, I hope?

> Since people don't want me to 

This too?

> can you please look into this issue and help me resolve this

Off-topic: if you want to stop contributing anyway, what is there to resolve? ;)

Please don't ping the app creator, he doesn't maintain Newpipe anymore.

> Thanks for your valuable time and effort. :D

Thank _you_ for your valuable time and effort. We're all spending our time and effort on this project, and would love for you to keep being part of the community. Come hang out with us on IRC. Discuss all the Newpipe or non-Newpipe issues you want. It's all welcome. But the 'punching people' has to stop.
